pepper-apis/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java

```diff
@@ -188,6 +188,7 @@ public PubsubMessage createMessage(String ddpMessageId,
                         queuedNotificationDto.getNotificationType(),
                         queuedNotificationDto.getNotificationServiceType(),
                         queuedNotificationDto.getTemplateKey(),
+                        queuedNotificationDto.isDynamicTemplate(),
```

Comment:
After this is updated with PR #233, this flag should come from `NotificationTemplate`.